spec/features/admin_adding_kudos_to_employees_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Admin adding kudos to employee', type: :feature do
+  let!(:admin) { create(:admin) }
+  let!(:employee1) { create(:employee, number_of_available_kudos: 10) }
+  let!(:employee2) { create(:employee, number_of_available_kudos: 10) }
```

Comment:
You can make `number_of_available_kudos` for employee1 and employee2 different to make the test more differential

Reply:
Changed 🙂